packages/elements-core/src/components/TryIt/Response/Response.spec.tsx

```diff
@@ -12,7 +12,7 @@ describe('Response', () => {
   window.URL.createObjectURL = jest.fn();
 
   afterEach(() => {
-    jest.resetAllMocks();
+    (window.URL.createObjectURL as jest.Mock).mockReset();
```

Comment:
I am not **really** sure why this change was needed.

What I understand is that mosaic uses some library that utilises https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver

Jest (or js-dom???) auto mocks this ResizeObserver, but then `resetAllMocks` for some reason totally nukes instances of that observer, causing tests to fail. 🤷

Reply:
Not sure if that's what you're referring to but we're using ResizeObserver in our tests setup:
https://github.com/stoplightio/elements/blob/7fc0c80ead2b9e1e6cba3023579f38e6b916a990/setupTests.ts?plain=1#L24

Reply:
So perhaps that is the reason why resetAllMocks nukes it. But it didn't break before, right? Anyway if that makes it work I think that's good enough. Though would be nice to know what's really happening there.

Reply:
`+1` to what Jakub says. Please create a follow-up issue for that, and let's merge what we have here :)